src/main/java/reactor/netty/http/client/HttpClient.java

```diff
@@ -839,7 +853,7 @@ public final HttpClient metrics(boolean metricsEnabled) {
 
 			return tcpConfiguration(tcpClient ->
 					tcpClient.bootstrap(b -> BootstrapHandlers.updateMetricsSupport(b,
-							MicrometerHttpClientMetricsRecorder.INSTANCE)));
+							new MicrometerHttpClientMetricsRecorder(maxUriTags))));
```

Comment:
The `INSTANCE` constant is no longer used and could be removed. Or it could still be applied when `maxUriTags == Metrics.MAX_URI_TAGS` but these methods aren't called often enough to benefit from a constant.